app/charts/chart-config-ui-options.ts

```diff
@@ -235,13 +241,29 @@ export const chartConfigOptionsUISpec: ChartSpecs = {
         componentTypes: ["GeoShapesDimension"],
         allowOtherFieldComponents: true,
         filters: true,
+        options: [
+          {
+            field: "color",
+            type: "component",
+            componentTypes: ["NumericalMeasure"],
+            optional: false,
+          },
+        ],
       },
       {
         field: "symbolLayer",
         optional: true,
         componentTypes: ["GeoCoordinatesDimension", "GeoShapesDimension"],
         allowOtherFieldComponents: true,
         filters: true,
+        options: [
+          {
+            field: "color",
+            type: "component",
+            componentTypes: ["NumericalMeasure", "GeoShapesDimension"],
+            optional: true,
+          },
```

Comment:
Should/will there be a "size" field here ?

Reply:
Yes 👌